.github/mergify.yml

```diff
@@ -0,0 +1,16 @@
+---
+# see https://docs.mergify.com/configuration/
+pull_request_rules:
+  - name: "Update out-of-date ready PRs"
+    conditions:
+      - "-closed"
+      - "-conflict"
```

Comment:
I looked at https://docs.mergify.com/conditions/#conditions, and from docs, I'd read this condition as "and the pull request has a conflict with the base branch". Did I misunderstand it?

Reply:
From https://docs.mergify.com/conditions/#grammar

> The optional `-` prefix is equivalent to the not operator.

Reply:
Ah! I actually misread this and didn't notice `-`. And I guess by default, the conditions use `and`?

Reply:
Correct